static/js/components/downloads.js

```diff
@@ -22,7 +46,30 @@
 
     if (catId != catRef) {
       $(that).addClass('downloads__category--active')
+      console.log(cat);
       $(compiled({ 'catId': catId, 'catTitle': cat[0].family, 'downloads': cat })).insertAfter( container )
+
+      // Load the latest version for each device type listed in this category
+      cat.forEach(function(deviceType) {
+        getLatestVersion(deviceType.id, function(err, data) {
+          var $link = $('a.js-download-link[data-device-id="' + deviceType.id + '"]');
+          if ($link) {
```

Comment:
This is a composite object and will always have a value and the branch will be taken.
Please change to `if ($link.length)`.